src/devices/Ssd1351/Ssd1351.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!--Disabling default items so samples source won't get build by the main library-->
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Command\*.cs" />
+    <Compile Include="*.cs" />
+    <None Include="README.md" />
+    <PackageReference Include="System.Device.Gpio" Version="$(SystemDeviceGpioPackageVersion)" />
+    <PackageReference Include="System.Drawing.Common" Version="4.6.0-preview4.19164.7" />
```

Comment:
NIT: you don't have to do this now, but I will later make sure that we move the version of System.Drawing.Common into Versions.props so that all of our bindings refer to the same version of it.

Reply:
OK. Please be aware the other bindings also use System.Common.Drawing.